splunk_add_on_ucc_framework/templates/globalConfig.json.init-template

```diff
@@ -49,6 +49,11 @@
                     ],
                     "title": "Accounts"
                 },
+                {%- if need_proxy %}
+                {
+                    "type": "proxyTab"
+                },
+                {%- endif %}
```

Comment:
Let's move it after loggingTab definition.

Reply:
Updated the template file.